src/registry_schemas/schemas/correction.json

```diff
@@ -170,6 +170,64 @@
               }
             }
           }
+        },
+        {
+          "if": {
+            "required": ["legalType"],
+            "properties": {
+              "legalType": {
+                "enum": [
+                  "CP"
+                ]
+              }
+            }
+          },
+          "then": {
+            "required": [
+              "contactPoint",
+              "business"
+            ],
+            "properties": {
```

Comment:
so `nameRequest` doesn't need to be one of the properties?

Reply:
just added.